src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -63,6 +52,32 @@ using namespace chip::app::Clusters;
       {{/last}}
     {{/chip_cluster_command_arguments}}
 
+    {{#unless mustUseTimedInvoke}}
```

Comment:
Why?  Why wouldn't we allow sending a group command for a command that requires timed invoke (and have it just fail), as opposed to ignoring the "is group" thing and trying to send it as a unicast command below?

Alternately, why wouldn't we just fail out explicitly if must use timed invoke and isGroupCluster?

Reply:
No good reason. Let me generates the code all the time.